did_resolver_registry/src/lib.rs

```diff
@@ -0,0 +1,228 @@
+pub mod error;
+
+use std::collections::HashMap;
+
+use did_resolver::{
+    did_parser::ParsedDID,
+    error::GenericError,
+    traits::resolvable::{
+        resolution_options::DIDResolutionOptions, resolution_output::DIDResolutionOutput,
+        DIDResolvable,
+    },
+};
+use error::DIDResolverRegistryError;
+
+pub struct ResolverRegistry {
+    resolvers: HashMap<String, Box<dyn DIDResolvable>>,
+}
```

Comment:
I'm just thinking that we could perhaps localize this to where it would be used in the beginning rather than have a completely different crate for it. It seems to me that the workspace is getting quite crowded, but it might just be me. @Patrik-Stas what do you think?

Reply:
Let's leave it here as discussed.